net/WebSocketHandler.hpp

```diff
@@ -674,7 +674,8 @@ class WebSocketHandler : public ProtocolHandlerInterface
             LOG_WRN("CheckTimeout: Timeout websocket: Ping: last " << _pingMicroS.last() << "us, avg "
                     << _pingMicroS.average() << "us >= " << _pingTimeout.count() << "us over "
                     << (int)_pingMicroS.duration() << "s");
-            shutdownSilent();
+            if( isIPType() )
```

Comment:
> Would this disconnect the internal connection between coolwsd and kit?

yes, internal connections are of type unix to the LocalServerSocket, validated.

Reply:
Re net::Default configuration:
- Its configuration via `configNet` UT callback  was added in parallel since it is beyond the WSD scope,
  where the general WSD configuration resides.
- Lack of thread sync is due to its lifecycle the same reason as WSD configuration, 
  i.e. we call `configNet` pre-creation of sockets etc.
- It is correct that we _had_ duplication of maximum sockets (ahem), which will be  revised due to changed semantics. However, the build global will influence this metric. Not too great, I agree.

Reply:
WIP: Dropping the net::Defaults instance and `configNet` (using `configure`) as requested.